pages/understanding-json-schema/reference/combining.md

```diff
@@ -135,6 +135,9 @@ Multiple of *both* 5 and 3 is rejected.
 15
 ```
 
+Careful consideration should be taken when using `oneOf` entries as the nature of it requires verification of *every* sub-schema
+which can lead to increased processing times. Prefer `anyOf` where possible.
```

Comment:
```suggestion
<Warning>
Careful consideration should be taken when using `oneOf` entries as the nature of it requires verification of *every* sub-schema which can lead to increased processing times. Prefer `anyOf` where possible.
</Warning>
```

Reply:
@unguul please proceed with this change and we are ready to go

Reply:
@unguul one change remaining, then this can be merged.